ci/Dockerfile.ci-jammy

```diff
@@ -1,13 +1,13 @@
-FROM ubuntu:focal-20200423
+FROM ubuntu:jammy-20230605
```

Comment:
Current is [jammy-20230816](https://hub.docker.com/layers/library/ubuntu/jammy-20230816/images/sha256-b492494d8e0113c4ad3fe4528a4b5ff89faa5331f7d52c5c138196f69ce176a6?context=explore)